event/event_target_bus.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+//go:build js
```

Comment:
question: Why this restriction?  And do we need an inverse restriction on `channelBus`?

Reply:
It's required to use the `syscall/js` package. The `channelBus` works fine in WASM so I don't see a need to restrict it.